web/client/actions/catalog.js

```diff
@@ -190,9 +193,10 @@ function addLayerAndDescribe(layer) {
                 if (results) {
                     let description = find(results, (desc) => desc.name === layer.name );
                     if (description && description.owsType === 'WFS') {
+                        const filteredUrl = ConfigUtils.filterUrlParams(ConfigUtils.normalizeUrl(description.owsURL), authkeyParamNameSelector(state));
```

Comment:
what's the different between this and the removeParameters? Maybe you can clarify the difference by documenting them in ConfigUtils.

Reply:
this removeParameters returns only a shrinked url without the skip params.
i'll change this name in order to clarify that.